richardltyler/vueniverse
# Feature/api calls

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- uses data fetched from the api instead of dummy data 

## *Instructions for reviewer:*
*Where should the reviewer start?*
- app.vue

*How should this be manually tested?*
- load the app and see that the displayed photo has today's date on it 

*Any background context you want to provide?*
*What are the relevant tickets?*
closes #2 #3

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*



## Checklist:
